docs/develop/make-office-web-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Make your Office Web Add-in compatible with an existing COM add-in-in
```

Comment:
```suggestion
title: Make your Office Add-in compatible with an existing COM add-in-in
```

Reply:
When referring to the Office Add-in, it's capital A, no "web." You can use lowercase "add-in" or  "web add-in" to refer to the same thing, second mention in a paragraph or sentence.